accounts/service_test.go

```diff
@@ -202,43 +200,40 @@ func TestAccountService(t *testing.T) {
 	testCases := []struct {
 		name  string
 		setup func(t *testing.T, lnd *mockLnd, r *mockRouter,
-			s *InterceptorService)
+			s *InterceptorService) []AccountID
 		startupErr string
 		validate   func(t *testing.T, lnd *mockLnd, r *mockRouter,
-			s *InterceptorService)
+			ids []AccountID, s *InterceptorService)
 	}{{
 		name: "startup err on invoice subscription",
 		setup: func(t *testing.T, lnd *mockLnd, r *mockRouter,
-			s *InterceptorService) {
+			s *InterceptorService) []AccountID {
 
 			lnd.invoiceSubscriptionErr = testErr
+
+			return nil
 		},
 		startupErr: testErr.Error(),
 		validate: func(t *testing.T, lnd *mockLnd, r *mockRouter,
-			s *InterceptorService) {
+			ids []AccountID, s *InterceptorService) {
 
 			lnd.assertNoInvoiceRequest(t)
 			require.False(t, s.IsRunning())
 		},
 	}, {
 		name: "err on invoice update",
 		setup: func(t *testing.T, lnd *mockLnd, r *mockRouter,
-			s *InterceptorService) {
-
-			acct := &OffChainBalanceAccount{
-				ID:             testID,
-				Type:           TypeInitialBalance,
-				CurrentBalance: 1234,
-				Invoices: AccountInvoices{
-					testHash: {},
-				},
-			}
+			s *InterceptorService) []AccountID {
 
-			err := s.store.UpdateAccount(ctx, acct)
+			acct, err := s.store.NewAccount(
+				ctx, 1234, testExpiration, "",
+			)
```

Comment:
we don't want to add the invoice here, right (and in the next test)?

Reply:
didnt add that cause it doesnt affect the tests/isnt what is being tested. 

but re-added it for consistency and easier review 👍